lib/utility/src/WatchdogWrapper.cpp

```diff
@@ -0,0 +1,21 @@
+
+#include "WatchdogWrapper.h"
+
+#include "mbed.h"
+
+namespace Utility {
+
+void WatchdogWrapper::startWatchdog(uint32_t countdown_ms /*= 5000*/, std::chrono::milliseconds pet_ms /*= 1000ms*/) {
+  Watchdog &watchdog = Watchdog::get_instance();
+  watchdog.start(countdown_ms);
+  Thread thread;
+  thread.start(callback(&WatchdogWrapper::petWatchdog, &pet_ms));
+}
+
+void WatchdogWrapper::petWatchdog(std::chrono::milliseconds *pet_ms) {
```

Comment:
pet_ms doesn't have to be a pointer type, just have the parameter type as `std::chrono::milliseconds`